Source/Core/Math.js

```diff
@@ -429,7 +427,6 @@ CesiumMath.DEGREES_PER_RADIAN = 180.0 / Math.PI;
  *
  * @type {Number}
  * @constant
- * @default {@link CesiumMath.RADIANS_PER_DEGREE} / 3600.0
```

Comment:
I'm okay with these changes, but it made me realize there's a bug with Math/CesiumMath in my PR, so thanks!

Reply:
I'm not positive this would work even without the Math error, because I can't find anything saying that an *expression* (not a *literal*) for `@default` is supposed to be supported.